tests/contract_testcases/solana/now.sol

```diff
@@ -0,0 +1,12 @@
+contract c {
+	struct S { int f1; }
+	function f(S s) public returns (int) {
+		return now.s;
+	}
+	function g() public {
+		now();
+	}
+	function h() public returns (uint) {
+		return now;
+	}
```

Comment:
Am I allowed to declare a variable named `now`?
```solidity
function j() public returns (int) {
    int now = 5;
    return now;
}
```

Reply:
Good point. I've added this test